types/openpgp/index.d.ts

```diff
@@ -39,6 +40,8 @@ export interface EncryptOptions {
 export interface EncryptedMessage {
     data?: string,
     message?: message.Message,
+    signature?: String |ReadableStream<String> |Signature // TODO add NodeStream
```

Comment:
This causes the build failure as well, you can have `ReadableStream<String>` in `ts3.2/` but not in ts3.1, so you can make this change in the `ts3.2` dir and in this file make it just `ReadableStream`

Reply:
@errietta done 👍